plugins/destination/postgresql/client/write.go

```diff
@@ -66,7 +83,11 @@ func (c *Client) Write(ctx context.Context, tables schema.Tables, res <-chan *de
 		if c.spec.WriteMode == specs.WriteModeAppend {
 			sql = c.insert(table)
 		} else {
-			sql = c.upsert(table)
+			if len(table.PrimaryKeys()) > 0 {
```

Comment:
IMO it's always true (no PK -> use `_cq_id`, see [here](https://github.com/cloudquery/plugin-sdk/blob/main/plugins/source/plugin.go#L75))

Reply:
now we add the `unique not null` to `_cq_id`. Do we want it to be just that (constraint on the column, no PK?)